site/public/js/simple-grading.js

```diff
@@ -361,6 +361,37 @@ function setupCheckboxCells() {
     $('td.cell-grade').click(function () {
         updateCheckpointCells(this);
     });
+    $('.cell-grade').change(function () {
+        let elem = $(this);
+        const split_id = elem.attr('id').split('-');
+        const row_el = $(`tr#row-${split_id[2]}-${split_id[3]}`);
+        const scores = {};
+        const old_scores = {};
+        row_el.find('.cell-grade').each(function () {
+            elem = $(this);
+            if (this.tagName.toLowerCase() === 'textarea') {
+                old_scores[elem.data('id')] = `${elem.data('origval')}`;
+                scores[elem.data('id')] = elem.val();
+                elem.data('origval', elem.val());
+                elem.attr('data-origval', elem.val());
+            }
+        });
+        submitAJAX(
+            buildCourseUrl(['gradeable', row_el.data('gradeable'), 'grading']),
+            {
+                csrf_token: csrfToken,
+                user_id: row_el.data('user'),
+                old_scores: old_scores,
+                scores: scores,
+            },
+            () => {
+                console.log('Success');
+            },
+            () => {
+                console.log('Failed to save data');
```

Comment:
Is there any additional information you can add to this message to make it easier to determine what went wrong if someone sends the contents of their browser console to a developer when something goes wrong?